lib/apr/views/commerce/commerce_order_slack_view.ex

```diff
@@ -24,6 +24,10 @@ defmodule Apr.Views.CommerceOrderSlackView do
       {"high_risk", "approved", %{"items_total_cents" => cents}}
         when cents >= 10_000_00 ->
           generate_slack_message(event, routing_key)
+      {"high_risk_async_payment", "processing_approval", %{"items_total_cents" => cents}}
+        when cents >= 10_000_00 ->
+          # require IEx; IEx.pry
```

Comment:
let's remove this one!

Reply:
Oh, yeah! Thank you for pointing it out. It's removed on this commit: https://github.com/artsy/aprd/pull/375/commits/53e306ad9c0532fd15f77750146f6524d4d22076